testsuite/shared/src/main/java/org/jboss/as/test/deployment/DeploymentInfoUtils.java

```diff
@@ -0,0 +1,781 @@
+         */
+        public String getCommand() {
+            return command;
+        }
+
+        /**
+         * Return joined string of parsed rows or output empty mark in case output is empty
+         *
+         * @return Return joined string of parsed rows or output empty mark in case output is empty
+         */
+        @Override
+        public String toString() {
+            if (isOutputEmpty()) {
+                return "<EMPTY>";
+            }
+            return String.join(LINE_SEPARATOR, rows);
+        }
+
+        /**
+         * If is set selected server group
+         *
+         * @return If server group has set return true, else false
+         */
+        public boolean hasServerGroup() {
+            return serverGroup != null;
+        }
+
+        /**
+         * Get selected server group in list/info command
+         *
+         * @return Selected server group in list/info command
+         */
+        public String getServerGroup() {
+            return serverGroup;
+        }
+
+        /**
+         * Get information message about selected server group.
+         *
+         * @return Information message about selected server group
+         */
+        public String getServerGroupInfo() {
+            return serverGroup != null ? " for server group '" + serverGroup + "'" : "";
+        }
+
+        /**
+         * Get original command output.
+         *
+         * @return Raw command output.
+         */
+        public String getOriginalOutput() {
+            return originalOutput;
+        }
+
+        // &&&& BEGIN Method for additional information about processing verify check
+
+        /**
+         * Additional information about processing verify check.
+         *
+         * @param request Request of management operations to verify check.
+         */
+        public void setRequest(String request) {
+            this.request = request;
+            response = null;
+        }
+
+        /**
+         * Additional information about processing verify check.
+         *
+         * @return Request of management operations to verify check.
+         */
+        public String getRequest() {
+            return request;
+        }
+
+        /**
+         * Additional information about processing verify check.
+         *
+         * @param response Response of management operations to verify check.
+         */
+        public void setResponse(String response) {
+            this.response = response;
+        }
+
+        /**
+         * Additional information about processing verify check.
+         *
+         * @return Response of management operations to verify check.
+         */
+        public String getResponse() {
+            return response;
+        }
+
+        // &&&& END   Method for additional information about processing verify check
+    }
+
+    /**
+     * Container of Utils internal mainer or optional parameters, reduce of overloaded function of check
```

Comment:
Sorry to bother you with english (I am also doing some mistakes...) but some sentences are still quite hard to understand. For example:
"Container of Utils internal mainer or optional parameters, reduce of overloaded function of check"
requires re-phrasing.